kube/services/frontend-framework/frontend-framework-deploy.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend-framework-deployment
+spec:
+  selector:
+    matchLabels:
+      app: frontend-framework
+  revisionHistoryLimit: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 2
+      maxUnavailable: 25%
+  template:
+    metadata:
+      labels:
+        app: frontend-framework
+        public: "yes"
+        GEN3_DATE_LABEL
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - frontend-framework
+              topologyKey: "kubernetes.io/hostname"
+      automountServiceAccountToken: false
+      volumes:
+      - name: ca-volume
+        secret:
+          secretName: "service-ca"
+      - name: config-volume
+        secret:
+          secretName: "frontend-framework-config"
+      - name: sponsor-img-volume
+        secret:
+          secretName: "frontend-framework-sponsor-config"
+      - name: privacy-policy
+        configMap:
+          name: "privacy-policy"
+      - name: cert-volume
+        secret:
+          secretName: "cert-portal-service"
+      containers:
+      - name: frontend-framework
+        GEN3_FRONTEND-FRAMEWORK_IMAGE
+        readinessProbe:
+          httpGet:
+            path: /ff/
+            port: 3000
+          initialDelaySeconds: 30
+          periodSeconds: 60
+          timeoutSeconds: 30
+        livenessProbe:
+          httpGet:
+            path: /ff/
+            port: 3000
+          initialDelaySeconds: 60
+          periodSeconds: 60
+          timeoutSeconds: 30
+          failureThreshold: 6
+        resources:
+          requests:
+            cpu: 0.6
+            memory: 512Mi
+          limits:
+            cpu: 2
+            memory: 4096Mi
+        ports:
+        - containerPort: 3000
+        command: 
+          - npm
+          - run
+          - start
+        env:
```

Comment:
mount `BASE_PATH` env var from here instead of in the dockerfile?

Reply:
Just seeing this now - good point, I'll switch this